### DISCUSSION TITLE
How to increase Wake LCD timeout?

### DISCUSSION THREAD

**Discussion by tensiondriven** [toverainc/willow#217] [Q&A]
Hola,

I got my ESP-BOX running with the Alexa wake world, hosted inference server, and Home Assistant fairly easily.  Overall the process was pretty painless! (One detail that would be nice to add to the README is that "Hey Lexin" is designed for Chinese users, this confused me)

Anyhoo, main point of this question:

After a couple of seconds, the ESP's display turns off.  This is altogether too short for me, and I'd prefer something closer to 10 seconds.

I see this in the logs:

```
I (01:27:26.458) WILLOW: Wake LCD timeout, turning off LCD
```

however I can't find anything about how to increase this timeout.

Is it possible to increase/change the Wake LCD timeout?

**Comment by hamishcunningham** [comment]
Looks like it is defined in DISPLAY_TIMEOUT_US in timer.h

**Comment by kristiankielhofner** [comment]
Where are working on dynamic configuration and this is likely something we can add relatively easily.

**Comment by Takore2002** [comment]
Huh, I didn't know I was supposed to time out, mine has yet to turn off after being on for a week...

**Reply by kristiankielhofner** [reply]
Can you create an issue for this? That shouldn't be happening!
